usr/src/lib/libzfs/common/libzfs_mount.c

```diff
@@ -1166,6 +1176,14 @@ mountpoint_cmp(const void *arga, const void *argb)
 	const char *a = mounta;
 	const char *b = mountb;
 	boolean_t gota, gotb;
+	uint64_t zoneda, zonedb;
+
+	zoneda = zfs_prop_get_int(za, ZFS_PROP_ZONED);
+	zonedb = zfs_prop_get_int(zb, ZFS_PROP_ZONED);
+	if (zoneda && !zonedb)
+		return 1;
```

Comment:
Cstyle nit -- `return (1);`

Reply:
Thanks - always forget that strange convention. I've just run pbchk and these were the only two things reported - fixed and pushed.